docs(dart-dio): add example for dio configuration (#13612)
getsentry/sentry-docs: includes/dart-integrations/dio.mdx

## Configure

To instrument Dio with Sentry, call `addSentry()` on your Dio instance as the very last step. 

> Perform any other configuration e.g., adding interceptors, transformers, or a custom HTTP client adapter before `addSentry()`.

Reviewer: ```suggestion
Perform any other configurations such as adding interceptors, transformers, or a custom HTTP client adapter before calling `addSentry()`.
```
